components/Layout.tsx

```diff
@@ -1,21 +1,27 @@
 import styled from 'styled-components';
+import Link from 'next/link';
 
 const Container = styled.div`
-  height: 100%;
+  min-height: 100%;
   display: flex;
   flex-direction: column;
 `;
 
 const Header = styled.header`
   padding: 1rem 2rem;
-  background-color: ${({ theme }) => theme.color.sunRay};
+  background-color: ${({ theme }) => theme.color.cream};
   font-size: 1.5rem;
   font-weight: 700;
+
+  a {
+    font-size: 1.5rem;
+    font-weight: 700;
+  }
```

Comment:
Not sure whom code this is. But is it not possible to do `const Link = styled(Link)` and style it inside directly? I am not a fan of this generic selector construct. Try to google it, if not possible - leave it as is

Reply:
```const Link = styled(Link)``` this doesn't work even tho according to some posts it should. I changed it anyway, but not sure if I improved it or not.